maintainers/maintainer-list.nix

```diff
@@ -16598,6 +16598,12 @@
     githubId = 9413924;
     name = "Thorsten Weber";
   };
+  twesterhout = {
```

Comment:
Would you please do this in a separate commit titled `maintainers: add twesterhout`? Thanks!

Reply:
Good point! Done.